docs/source/index.rst

```diff
@@ -28,12 +28,14 @@ application.
 
 
 
-How can I build it?
--------------------
+Index
+-----
 
 .. toctree::
    :maxdepth: 2
 
+   introduction.rst
+
```

Comment:
I would not add a blank line between sections.

Reply:
new pull request for documentation